src/views/product-downloads-view/index.tsx

```diff
@@ -19,13 +20,21 @@ import FeaturedTutorialsSection from './components/featured-tutorials-section'
 import OfficialReleasesSection from './components/official-releases-section'
 import s from './product-downloads-view.module.css'
 
+// exclude pre-releases and such
+const VALID_SEMVER_REGEX = /^\d+\.\d+\.\d+$/
+
 const ProductDownloadsView = ({
   latestVersion,
   pageContent,
   releases,
 }: ProductDownloadsViewProps): ReactElement => {
   const versionSwitcherOptions = useMemo(() => {
-    return semverRSort(Object.keys(releases.versions)).map((version) => {
+    return semverRSort(
+      Object.keys(releases.versions).filter((version) => {
+        const isValidRegex = !!version.match(VALID_SEMVER_REGEX)
+        return isValidRegex
+      })
+    ).map((version) => {
```

Comment:
These changes will go away once #140 has been merged and this branch is updated.

Reply:
PR merged and branch updated 😊